### PR TITLE
operator: split operator.go into several files

### PR DESCRIPTION
**Pull request by Luffbee**
### What problem does this PR solve? 

`operator.go` is too big. It contains several structs.

### What is changed and how it works?

* Move `OpStep` and its implementations into `step.go`.
* Move `OpInfluence` and `StoreInfluence` into `influence.go`.
* Rename `operator_kind.go` to `kind.go`.

### Check List 

Tests 

 - No code

### CODE REVIEW

File: server/schedule/operator/operator.go [hunk 1]
```diff
@@ -1144,3 +738,28 @@ func CheckOperatorValid(op *Operator) bool {
 	}
 	return true
 }
+
+type u64Set map[uint64]struct{}
```
Comment (lhy1024) [inline server/schedule/operator/operator.go]: why do we need it

Comment (disksing) [reply server/schedule/operator/operator.go]: for formatting

**Comment by disksing** [comment]
Cool. I was thinking about doing the same thing yesterday :)

File: server/schedule/operator/operator.go (hunk 1 above)
Comment (disksing) [reply server/schedule/operator/operator.go]: better move to L757

**Comment by disksing** [comment]
How about moving `Create*` functions to another file too?

**Comment by Luffbee** [comment]
> How about moving `Create*` functions to another file too?

Moved them to `create_operator.go`.

**Pull request merged**